### PR TITLE
Feature/entity sets with ids

### PR DESCRIPTION
**Pull request by btiernay**

### CODE REVIEW

File: dcc-portal-api/src/main/java/org/icgc/dcc/portal/model/EntitySet.java [hunk 1]
```diff
@@ -144,10 +148,7 @@ public static EntitySet createForStatusFinished(final UUID id, final String name
   @Getter
   @ApiModel(value = "State")
   public enum State {
-    PENDING("pending"),
-    IN_PROGRESS("in progess"),
-    FINISHED("finished"),
-    ERROR("error");
+    PENDING("pending"), IN_PROGRESS("in progess"), FINISHED("finished"), ERROR("error");
```
Comment (andricDu) [inline dcc-portal-api/src/main/java/org/icgc/dcc/portal/model/EntitySet.java]: Looks like your eclipse didn't read the settings correctly.

Comment (btiernay) [inline dcc-portal-api/src/main/java/org/icgc/dcc/portal/model/EntitySet.java]: Ya, noticed that. Will have a look.

**Comment by btiernay** [comment]
retest this please

**Comment by btiernay** [comment]
retest this please

**Pull request merged**